docs/reference/examples/human_impact.json

```diff
@@ -0,0 +1,29 @@
+{
+  "namespace": "ssvc",
+  "version": "1.0.0",
+  "name": "Human Impact",
+  "description": "Human Impact",
+  "key": "HI",
+  "values": [
+    {
+      "name": "Low",
```

Comment:
Comment from @sei-vsarvepalli:

> calls the "values" output as "name" shouldn't this be labeled "value"? This we can disambiguate the "name" that is used for the Decision Point itself.

Reply:
My approach was to create a python base class (`ssvc._mixins._Base`) that just has `name` and `description` attributes. Both `SsvcDecisionPointValue` and `SsvcDecisionPoint` objects inherit from this `_Base`. I don't see a problem with classes consistently having a `name` attribute. It seems less desirable to me to have one object's name be in the `name` attribute and another object's `name` be in a `value` attribute. The class name already indicate that it's a `SsvcDecisionPointValue` so I think it would be unabmbiguous that an `SsvcDecisionPointValue.name` is the name of a decision point value whereas `SsvcDecisionPoint.name` is the name of a decision point.

Reply:
FWIW, it would make sense to me if the elements of "values" were "value" and not "name" but as long as the name space is unambiguous, it's OK.